modules/test/playwright/tests/portal-language-override-web/languageOverride.spec.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {languageOverridePageTest} from '../../fixtures/LanguageOverridePageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {TTranslation} from '../../pages/portal-language-override-web/LanguageOverridePage';
+import getRandomString from '../../utils/getRandomString';
+import {readFileFromZip} from '../../utils/zip';
+
+export const test = mergeTests(loginTest(), languageOverridePageTest);
+
+test('LPD-33373 assert that overriden translations can be exported', async ({
+	languageOverridePage,
+	page,
+}) => {
+	const translations: TTranslation[] = [
+		{
+			key: getRandomString(),
+			values: [
+				{
+					languageId: 'en-US',
+					value: getRandomString(),
+				},
+				{
+					languageId: 'es-ES',
+					value: getRandomString(),
+				},
+				{
+					languageId: 'pt-BR',
+					value: getRandomString(),
+				},
+			],
```

Comment:
@thiago-buarqque , this is different than what I was suggesting, but also works. I was suggesting this:
```
values: {
	['en-US']: getRandomString(),
	['es-ES']: getRandomString(),
	['pt-BR']: getRandomString(),
}
```

My way is a little less verbose but yours is more explicit. They both have pros and cons so feel free to choose what you think is best in this case.

Reply:
@ethib137 I prefer this way :sweat_smile: For me it's easier to understand. But thanks though